### ISSUE TITLE
KernelSHAP argmax prediction for binary classifiers with one output

### ISSUE THREAD

**Issue by jklaise** [SeldonIO/alibi#225]
Currently `KernelSHAP` computes the classification label using an `argmax`: https://github.com/SeldonIO/alibi/blob/7d7347392ed8dd212c0a33be6997847452b24860/alibi/explainers/shap_wrappers.py#L837

However, some binary classification models only output one scalar (e.g. score or probability of first class) in which case this would not work. The solution is already in the TreeSHAP PR (#206) so once that is merged we can update KernelSHAP.

**Comment by jklaise** [comment]
This needs a bit more thought as it can't be done in the same way as in TreeShap because the accepted (classification) models can output either probabilities of classes or logits and without the user telling us we cannot know which one it is.
